RealEstate/apps/core/views.py

```diff
@@ -296,6 +332,16 @@ def post(self, request, *args, **kwargs):
                             content_type="application/json")
 
 
+class PasswordChangeDoneView(BaseView):
+    """
+    This is needed to provide a message to the user that their password change
+    was successful.
+    """
+    def get(self, request, *args, **kwargs):
+        messages.success(request, "Password change successful")
```

Comment:
Maybe we can make this more personal? "You successfully changed your password." or "Congratulations! You may have done nothing else with your life, but at least you changed your password."